### ISSUE TITLE
Documentation is completely broken - it points to the develop branch but says it's for 0.20.1

### ISSUE THREAD

**Issue by PawelRoman** [tortoise/tortoise-orm#1603]
The official documentation is generated from the develop branch, but it says it's for the official 0.20.1 version

For example, take a look at this page: https://tortoise.github.io/contrib/fastapi.html

And this: https://tortoise.github.io/examples/fastapi.html

It makes this import `tortoise.contrib.fastapi.RegisterTortoise`

But in 0.20.1 there is no such class. 0.20.1 is using `register_tortoise` synchronous function which seems to be a breaking change in the library.

**Comment by waketzheng** [comment]
You install/upgrade the package from github to use the `RegisterTortoise` class:
```bash
pip install --upgrade git+https://github.com/tortoise/tortoise-orm
```
Or by ssh
```bash
pip install --upgrade git+ssh://git@github.com/tortoise/tortoise-orm
```

**Comment by waketzheng** [comment]
Documentation should be able to select version to fix this issue and #1569

**Comment by abondar** [comment]
Yeah, right now documentation publishing is wacky 

I'll try to fix it with release of 0.21.0, so it will deploy docs only in case of new release, not just new commits

Github pages doesn't support doc versioning automatically, but it seems out there are some plugins, that say that they can handle it, I'll look into it in mean time

**Comment by qingshuiyuyu** [comment]
i cant believe why the FastAPI Examples can not be run    
when i use register_tortoise
the error：`
async with register_tortoise(
AttributeError: __aenter__
`

**Comment by waketzheng** [comment]
@qingshuiyuyu Use `async with RegisterTortoise` instead. https://tortoise.github.io/examples/fastapi.html#main-py

**Comment by qingshuiyuyu** [comment]
> @qingshuiyuyu Use `async with RegisterTortoise` instead. https://tortoise.github.io/examples/fastapi.html#main-py

but i can not import this class v:tortoise-orm==0.20.1 ,there only register_tortoise
i use another way this guy rewrite the class  [https://github.com/tortoise/tortoise-orm/issues/1576#issuecomment-2074505541](url)

**Comment by waketzheng** [comment]
> Yeah, right now documentation publishing is wacky
> 
> I'll try to fix it with release of 0.21.0, so it will deploy docs only in case of new release, not just new commits
> 
> Github pages doesn't support doc versioning automatically, but it seems out there are some plugins, that say that they can handle it, I'll look into it in mean time

@abondar How is that?

**Comment by abondar** [comment]
Seems fixed now with release of 0.21.0
